src/util/CookieUtil.js

```diff
@@ -54,5 +59,26 @@ define(['okta', 'vendor/plugins/jquery.cookie'], function (Okta) {
     removeCookie(REMEMBER_DEVICE_COOKIE_NAME);
   };
 
+  fn.isAutoPushEnabled = function (userId) {
+    if (userId === undefined) {
+      return false;
+    }
+    return $.cookie(getAutoPushKey(userId)) === "true";
```

Comment:
single quotes - fyi we did notice yesterday that we weren't reporting lint failures in travis, but once you rebase you'll be able to see this fail in travis.